.github/workflows/deploy.yml

```diff
@@ -37,15 +37,16 @@ jobs:
           tar -xf iota.tar
           tar -xf shimmer.tar
           tar -xf next.tar
-      # Convert to Vercel project and upload
-      - name: Install Vercel CLI
-        run: npm install --global vercel@latest
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Build Project Artifacts
-        run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Deploy Project Artifacts to Vercel
-        run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
+      # Deploy to Vercel
+      - name: Deploy to Vercel Action
+        uses: BetaHuhn/deploy-to-vercel-action@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+          PR_LABELS: false
+          PREBUILT: true
```

Comment:
This is not going to work because neither `FORCE` or `PREBUILT` are in the [latest released version](https://github.com/BetaHuhn/deploy-to-vercel-action/blob/v1.9.12/action.yml#L5C1-L77) of this action.
And judging by the workflow it looks like it might be an important flag. Just letting you know.

Reply:
Good catch. Thanks for that. We can just wait for the next release or add the latest changes from the repo.
@jlvandenhout wdyt?

Reply:
Yeah, after the restructure we will have a much less complex build, so we might go back to a more vanilla Vercel workflow anyway.

Reply:
Ok. Will convert this to a DRAFT in the meantime. @VmMad thanks for your help ❤️